### PR TITLE
Added support for tvOS

### PR DESCRIPTION
**Pull request by simongranger**

### CODE REVIEW

**Comment by simongranger** [comment]
For node-simctl hardcoded to iOS #11892
https://github.com/appium/appium/issues/11892

**Comment by imurchie** [comment]
Fixing this is the intention of #72 as well.

File: lib/simctl.js [hunk 1]
```diff
@@ -445,24 +445,30 @@ async function getDevicesByParsing () {
...
-  // make an entry for each sdk version
-  while (match !== null) {
-    matches.push(match);
-    match = deviceSectionRe.exec(stdout);
-  }
-  if (matches.length < 1) {
-    log.errorAndThrow('Could not find device section');
-  }
+  let match;
+  deviceSectionRe.forEach(function (element) {
```
Comment (jlipps) [inline lib/simctl.js]: we use simple `for...let` loop construct instead of `.forEach`

**Comment by simongranger** [comment]
I will close this PR

**Pull request closed without merging**